.github/workflows/tests.yml

```diff
@@ -178,7 +178,7 @@ jobs:
       requiresTestSdk: true
 
   endtoend_tests:
-    name: EndToEnd ${{ matrix.os }}
+    name: EndToEnd Linux
```

Comment:
Assuming in the future we plan on adding the windows and osx matrix and then we'll revert this back?